hardware/gyros.py

```diff
@@ -29,6 +31,30 @@
 import json
 import re
 import threading
+import asyncio
+import random
+from hardware import u
+
+class MockGyro:
+    def __init__(self, instr_name = None):
+        if not instr_name:
+            self.name = "Gyroscope"
+        else:
+            self.name = instr_name
+        self.data = {"name": self.name,

```

Comment:
I recommend moving `"name: self.name,` to the next line to be consistent with other dict instantiations.